iree/turbine/kernel/wave/utils/print_utils.py

```diff
@@ -60,18 +61,32 @@ def try_apply_pass(
     trace: CapturedTrace,
     print_ir_before: Sequence[str] = [],
     print_ir_after: Sequence[str] = [],
+    pass_times: Optional[dict[str, float]] = None,
 ):
-    if "all" in print_ir_before or p.__name__ in print_ir_before:
-        print(f"***Before {p.__name__}***\n")
+    name = p.__name__
+    if "all" in print_ir_before or name in print_ir_before:
+        print(f"***Before {name}***\n")
         print_trace(trace)
     try:
+        start = timeit.default_timer()
         p()
+        end = timeit.default_timer()
+        if pass_times is not None:
+            if name not in pass_times:
+                pass_times[name] = end - start
```

Comment:
minor: pass_times[name] can be hoisted out of the if-else.

Reply:
done